languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt

```diff
@@ -151,3 +152,50 @@ Xbox
 Twitch
 iOS
 Alibaba
+concientizar
+UPS
+abuelita
+casita
+City
+Center
+River
+New
+mami
+papi
+Smith
+Hey
+AM
+PM
```

Comment:
https://spanish.stackexchange.com/questions/888/use-of-am-pm-in-time

not sure if this should be upper and/or down cased